radio/src/gps.cpp

```diff
@@ -207,12 +228,14 @@ bool gpsNewFrameNMEA(char c)
           switch (gps_frame) {
             case FRAME_GGA:
               frameOK = 1;
+              gpsData.fix = gps_Msg.fix;
+              gpsData.numSat = gps_Msg.numSat;
               if (gps_Msg.fix) {
-                gpsData.fix = 1;
+                __disable_irq();    // do the atomic update of lat/lon
```

Comment:
Currently it is not necessary, but if in future we move the gps code  in another thread it will be needed.